packages/frontend/src/features/MetricFlow/hooks/useMetricFlowQueryResultsAPI.ts

```diff
@@ -0,0 +1,99 @@
+import { ApiError } from '@lightdash/common';
+import { useQuery } from 'react-query';
+import { UseQueryOptions } from 'react-query/types/react/types';
+import {
+    createMetricFlowQuery,
+    CreateMetricFlowQueryResponse,
+    getMetricFlowQueryResults,
+    GetMetricFlowQueryResultsResponse,
+    QueryStatus,
+} from '../../../api/MetricFlowAPI';
+
+type ApiRequestsState = Pick<
+    ReturnType<typeof useQuery<GetMetricFlowQueryResultsResponse, ApiError>>,
+    'isLoading' | 'data' | 'error' | 'status'
+> &
+    Pick<
+        ReturnType<typeof useQuery<CreateMetricFlowQueryResponse, ApiError>>,
+        'refetch'
+    >;
+
+export const useMetricFlowQueryResultsAPI = (
```

Comment:
what's the naming convention from now for api calls? I thought `...API` was for the raw fetch queries, whereas this is state managed by react query